knora/dsplib/utils/xml_upload.py

```diff
@@ -351,42 +359,69 @@ def xml_upload(input_file: str, server: str, user: str, password: str, imgdir: s
         id2iri_mapping, failed_uploads = _upload_resources(resources, imgdir, sipi_server, permissions_lookup,
                                                            resclass_name_2_type, id2iri_mapping, con, failed_uploads)
     except BaseException as err:
-        _handle_upload_error(err, input_file, id2iri_mapping, failed_uploads, stashed_xml_texts, stashed_resptr_props)
+        _handle_upload_error(
+            err=err,
+            id2iri_mapping=id2iri_mapping,
+            failed_uploads=failed_uploads,
+            stashed_xml_texts=stashed_xml_texts,
+            stashed_resptr_props=stashed_resptr_props,
+            proj_shortcode=shortcode,
+            onto_name=default_ontology,
+            server=server,
+            save_location=save_location
+        )
 
     # update the resources with the stashed XML texts
-    nonapplied_xml_texts = {}
```

Comment:
It's not necessary any more to create an empty `nonapplied_xml_texts`, because the case cannot happen any more that `nonapplied_xml_texts` is used before being instantiated